src/components/modal/report/modal.jsx

```diff
@@ -124,120 +79,88 @@ class ReportModal extends React.Component {
-                                                )
-                                            }}
-                                        />
-                                    </div>
-                                    <Select
-                                        required
-                                        elementWrapperClassName="report-modal-field"
-                                        label={null}
-                                        name="report_category"
-                                        options={REPORT_OPTIONS.map(option => ({
-                                            value: option.value,
-                                            label: this.props.intl.formatMessage(option.label),
-                                            key: option.value
-                                        }))}
-                                        validationErrors={{
-                                            isDefaultRequiredValue: this.props.intl.formatMessage({
-                                                id: 'report.reasonMissing'
-                                            })
-                                        }}
-                                        value={this.state.category}
-                                        onChange={this.handleCategorySelect}
-                                    />
-                                    <TextArea
-                                        required
-                                        className="report-text"
-                                        elementWrapperClassName="report-modal-field"
-                                        label={null}
-                                        name="notes"
-                                        placeholder={this.lookupPrompt(this.state.category)}
-                                        validationErrors={{
-                                            isDefaultRequiredValue: this.props.intl.formatMessage({
-                                                id: 'report.textMissing'
-                                            }),
-                                            maxLength: this.props.intl.formatMessage({id: 'report.tooLongError'}),
-                                            minLength: this.props.intl.formatMessage({id: 'report.tooShortError'})
-                                        }}
-                                        validations={{
-                                            maxLength: 500,
-                                            minLength: 20
+                    <ModalInnerContent className="report-modal-content">
+                        {isError && (
+                            <div className="error-text">
+                                <FormattedMessage id="report.error" />
+                            </div>
+                        )}
+                        <Progression step={isConfirmed ? STEPS.confirmation : this.state.step}>
+                            <FormStep
```

Comment:
Could you put a comment before this line, making clear that it constitutes the content of the `category` step?

Reply:
Done